src/util/address.rs

```diff
@@ -381,17 +390,34 @@ impl Display for Address {
             } => {
                 let hrp = match self.network {
                     Network::Bitcoin => "bc",
-                    Network::Testnet | Network::Signet  => "tb",
+                    Network::Testnet | Network::Signet => "tb",
                     Network::Regtest => "bcrt",
                 };
```

Comment:
I believe this code belongs to `Network` type, since this logic may be re-used elsewhere

Reply:
I don't remember any code that needs to reuse same stringly network marker as used in addresses. Do you? If there's no such code then I believe this is the right place.

Reply:
Yes, or alternatively the constants crate if it ever is revived. But it isn't relevant to this PR imo.

Reply:
I am quite sure the same values are involved at least in address parsing

Reply:
Anyway I agree with the fact that this does not belong to this PR and should be a separate topic